invenio_app_ils/records/views.py

```diff
@@ -154,7 +164,31 @@ class AcceptRequestResource(ContentNegotiatedMethodView):
     @pass_record
     def post(self, pid, record, **kwargs):
         """Accept request post method."""
-        raise DocumentRequestError("Accept request is not implemented")
+        data = request.get_json() or dict()
```

Comment:
```suggestion
        data = self.loader()
```
Please fix all the views in this file.

Reply:
@ntarocco Thanks a lot I didn't manage to make it work yesterday with the loaders, this is probably what I was missing.